lib/pengine/clone.c

```diff
@@ -152,7 +152,12 @@ clone_unpack(pe_resource_t * rsc, pe_working_set_t * data_set)
     // Implied by calloc()
     /* clone_data->xml_obj_child = NULL; */
 
-    clone_data->clone_node_max = crm_parse_int(max_clones_node, "1");
+    // Use 1 as default but 0 for minimum and invalid
+    if (max_clones_node == NULL) {
+        clone_data->clone_node_max = 1;
+    } else {
+        pcmk__scan_min_int(max_clones_node, &clone_data->clone_node_max, 0);
```

Comment:
Does this not need to be `&(clone_data->clone_node_max)`?  I've seen that usage elsewhere, and I'm sure I have had to do it myself at some point.

Reply:
That's the default precedence, it's just clearer to make it explicit. Done.